roles/argocd/tasks/main.yml

```diff
@@ -37,6 +37,16 @@
         server:
           route:
             enabled: true
+          insecure: true
```

Comment:
This is the only line I added. Not sure why the diff is showing changes from my last PR